### ISSUE TITLE
Sprite loop

### ISSUE THREAD

**Issue by gpaixao** [raphamorim/origami.js#44]
I'm using `origami.js` for create a sample game, and I need my sprite in animation loop.
I saw the source code and understand that your first condition is if the sprite stay outside of canvas and after if the `loop` is false.

```js
function drawSprite(sprite) {
  var self = this;

  if (sprite.posX === sprite.widthTotal) {
    if (sprite.loop === false) {
      window.cancelAnimationFrame(sprite.animation);
      return;
    }
    sprite.posX = 0;
  }

...
```

I think that logic is:

```js
function drawSprite(sprite) {
  var self = this;

  if (sprite.loop === false) {
      window.cancelAnimationFrame(sprite.animation);
      return;
    }

  if (sprite.posX === sprite.widthTotal ) {
    sprite.posX = 0;
  }

...
```

If I want create a true loop, where my sprite forever animate.

The `origami.js`just execue one time the animation and not work like a loop.

*Example:*

![example .gif](https://cms-assets.tutsplus.com/uploads/users/358/posts/25363/image/00-final-product-final.gif)

**Comment by raphamorim** [comment]
I see, can you send a PR with theses adjustments?

**Comment by raphamorim** [comment]
Sorry I didn't understand.
Infinite loops actually works [see it here](http://origamijs.com/docs/#sprite). Your issue is about finite loops?

**Comment by raphamorim** [comment]
Btw: Nice example, if you want add other animation examples in [origamijs docs](https://github.com/raphamorim/origamijs.com) :D

**Comment by gpaixao** [comment]
I've a question. 

```js
Origami.sprite = function(x, y, properties) {
  var self = this;

  if (!properties || !properties.src)
    return this;

  var image = new Image(),
    frames = (properties.frames || 0), //here
    loop = (properties.loop || true), //here
    speed = (properties.speed || 10); //here
```

You set `frames`, `loop` and `speed`, but where you use?

**Comment by raphamorim** [comment]
Nice!
Closed in #53
